packages/app/src/app/components/CodeEditor/ImageViewer/index.tsx

```diff
@@ -1,24 +1,25 @@
-// @flow
-import React from 'react';
+import * as React from 'react';
 import Input from 'common/components/Input';
 import Button from 'app/components/Button';
 import { Container, Title, SubTitle, Image, MaxWidth } from './elements';
+import { Props as CodeEditorProps } from '../';
 
-import type { Props, Editor } from '../types';
+type Props = CodeEditorProps & {
+  dependencies: any;
+};
 
-export default class ImageViewer extends React.Component<Props>
-  implements Editor {
```

Comment:
I think we lose the autocompletion of functions of the editor of we remove this implements.

Reply:
Ah, yeah, I thought it was unlikely to change again. But yeah, better to have it there. Actually ImageViewer does not implement anything from Editor, so get lint error there. But put it back on Codemirror and Monaco :)